tardis/montecarlo/montecarlo_numba/estimators.py

```diff
@@ -65,24 +177,24 @@ def update_bound_free_estimators(
         photo_ion_rate_estimator_increment = (
             comov_energy * distance * x_sect_bfs[i] / comov_nu
         )
-        numba_estimator.photo_ion_estimator[
+        estimator_state.photo_ion_estimator[
```

Comment:
Should this section be using the increment method instead?

Reply:
no, the increment method is just adding two estimator classes from two different threads. However, a much more detailed restructure of this is done in #2500 where we will document it.